ios/FluentUI/People Picker/AvatarViewVnext.swift

```diff
@@ -0,0 +1,340 @@
+            borderRadius = appearanceProxy.borderRadius.xlarge
+            ringThickness = appearanceProxy.ringThickness.xlarge
+            ringInnerGap = appearanceProxy.ringInnerGap.xlarge
+            ringOuterGap = appearanceProxy.ringOuterGap.xlarge
+            presenceIconSize = appearanceProxy.presenceIconSize.xlarge
+            presenceIconOutlineThickness = appearanceProxy.presenceIconOutlineThickness.xlarge
+            textFont = appearanceProxy.textFont.xlarge
+        case .xxlarge:
+            avatarSize = appearanceProxy.size.xxlarge
+            borderRadius = appearanceProxy.borderRadius.xxlarge
+            ringThickness = appearanceProxy.ringThickness.xxlarge
+            ringInnerGap = appearanceProxy.ringInnerGap.xxlarge
+            ringOuterGap = appearanceProxy.ringOuterGap.xxlarge
+            presenceIconSize = appearanceProxy.presenceIconSize.xxlarge
+            presenceIconOutlineThickness = appearanceProxy.presenceIconOutlineThickness.xxlarge
+            textFont = appearanceProxy.textFont.xxlarge
+        }
+    }
+}
+
+/// View that represents the avatar
+public struct AvatarVnextView: View {
+    @ObservedObject var tokens: AvatarTokens
+    @ObservedObject var state: AvatarVnextState
+
+    public init(style: AvatarVnextStyle,
+                size: AvatarVnextSize) {
+        self.tokens = AvatarTokens(style: style, size: size)
+        self.state = AvatarVnextState()
+    }
+
+    public var body: some View {
+        let avatarImageSize: CGFloat = tokens.avatarSize!
+        let ringInnerGapSize: CGFloat = avatarImageSize + (tokens.ringInnerGap * 2)
+        let ringSize: CGFloat = ringInnerGapSize + (tokens.ringThickness * 2)
+        let ringOuterGapSize: CGFloat = ringSize + (tokens.ringOuterGap * 2)
+        let presenceIconSize: CGFloat = tokens.presenceIconSize!
+        let presenceIconOutlineSize: CGFloat = presenceIconSize + (tokens.presenceIconOutlineThickness * 2)
+
+        // Calculates the positioning of the presence icon ensuring its center is always on top of the avatar circle's edge
+        let ringInnerGapRadius: CGFloat = (ringInnerGapSize / 2)
+        let ringInnerGapHypotenuse: CGFloat = sqrt(2 * pow(ringInnerGapRadius, 2))
+        let presenceIconHypotenuse: CGFloat = sqrt(2 * pow(presenceIconOutlineSize / 2, 2))
+        let presenceFrameHypotenuse: CGFloat = ringInnerGapHypotenuse + ringInnerGapRadius + presenceIconHypotenuse
+        let presenceIconFrameSide: CGFloat = sqrt(pow(presenceFrameHypotenuse, 2) / 2)
+
+        let ringColor = state.ringColor ?? tokens.ringDefaultColor!
+        let backgroundColor = state.backgroundColor ?? tokens.backgroundDefaultColor!
+        let foregroundColor = state.foregroundColor ?? tokens.foregroundDefaultColor!
+
+        let presence = state.presence
+        let style = tokens.style
+        let initialsString: String = ((style == .overflow) ? state.primaryText ?? "" : InitialsView.initialsText(fromPrimaryText: state.primaryText,
```

Comment:
shouldn't the initial's view 21 different background color be tokenized?

Reply:
Seems like we're going to have 30 default colors now. I'm following up with Victor to get this updated here.

Reply:
Create #338 to track this issue.